recipes/pixy/build.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+mkdir -p $PREFIX/bin
+cp $SRC_DIR/src/pixy.py $PREFIX/bin/pixy
+chmod +x $PREFIX/bin/pixy
```

Comment:
Could you please delete this file and also the ``bld.dat`` I think they are not necessary